presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -46,6 +46,7 @@
     private final SessionRepresentation session;
     private final QueryState state;
     private final MemoryPoolId memoryPool;
+    private final boolean queued;
```

Comment:
We don't need to add this to `QueryInfo`, since it already has `QueryState`. We do it for `scheduled` since determining that is a complex operation involving walking the stage tree.

We only need to add the flag to `StatementStats` since that's the public API for the client.